oaiharvest/harvest.py

```diff
@@ -73,11 +82,33 @@ class OAIHarvester(object):
     def __init__(self, mdRegistry):
         self._mdRegistry = mdRegistry
 
+    def pause(self, now, until):
+        """ Unconditionally pause the process from `now` to `until`. """
+        logger = logging.getLogger(__name__).getChild('OAIHarvester.pause')
+        logger.info('Pausing until {} (incremental harvest).'.format(until))
+        sleep((until - now) / timedelta(seconds=1))
+
+    def maybe_pause_if_incremental(self, time_range):
+        """ Pause the process depending on incremental time range settings. """
+        if time_range is None:
+            return
+        now = datetime.now()
+        start = datetime.combine(now.date(), time_range[0].time())
+        stop = datetime.combine(now.date(), time_range[1].time())
+        if now < start:
```

Comment:
Please could you add some tests for the method to validate this logic?

Reply:
Sure. It may take a while before I get to it, though.